projects/file-parsing/main_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	filesConverting "file-parsing/types-converting"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestToStruct(t *testing.T) {
+
+	files := []string{"examples/json.txt", "examples/repeated-json.txt", "examples/data.csv", "examples/custom-binary-le.bin", "examples/custom-binary-be.bin"}
+	var got []filesConverting.Score
+
+	t.Run("Checking error when can't read a file", func(t *testing.T) {
+		myError := errors.New("open amples/json.txt: no such file or directory")
+		_, err := readingFiles("amples/json.txt")
+
+		require.EqualError(t, err, myError.Error())
+	})
+
+	t.Run("Parsing different types to struct", func(t *testing.T) {
+		for _, file := range files {
```

Comment:
I would put a `t.Run` call inside this loop, so that each file is treated like a separate test here. Otherwise if one of the files is parsed wrong, the rest of the files won't be tested.